bin/diff_yaml.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+# Compare 2 multi document kubernetes yaml files
+# It ensures that order does not matter
+
+import yaml  # pyyaml
+import datadiff
```

Comment:
this would fail pep8 (imports should be sorted)

Reply:
I have run `autopep8 -i `

Reply:
hmm, maybe thats just a pep8 rule that isn't defaulted in pep8.  The pep standards change all the time.

Reply:
https://www.python.org/dev/peps/pep-0008/#imports - doesn't indicate sorting - perhaps that was changed, or has been a custom rule on projects in the past.